.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
- - "node"
+ - "12.10"
```

Comment:
Node JS version for Travis CI is set to `12.10`. Versions from `13.0` led to build errors as of this writing.